packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -136,7 +136,7 @@ export default baseMixins.extend<options>().extend({
       return this.isDirty || dirtyTypes.includes(this.type)
     },
     isSingle (): boolean {
-      return this.isSolo || this.singleLine || this.fullWidth
+      return this.isSolo || this.singleLine || this.fullWidth || (this.dense && !this.outlined)
```

Comment:
how come that dense affects isSingle?

Reply:
this to make label hidden on focus except outlined

![image](https://user-images.githubusercontent.com/8259014/64068912-39699880-cc37-11e9-8baa-a622523f161a.png)

from md website:
https://material.io/components/assets/11tg--XoPYHGk4sg1AF3yYOz3sVjQM4XP/textfields-filled-dense.png